tests/cpp-tests/proj.android-studio/gradle.properties

```diff
@@ -16,10 +16,18 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-PROP_COMPILE_SDK_VERSION=14
-PROP_MIN_SDK_VERSION=10
-PROP_TARGET_SDK_VERSION=22
-PROP_APP_PLATFORM=10
+
+# Android SDK version that will be used as the compile and target version of this project
+PROP_SDK_VERSION=27
+
+# Android SDK version that will be used as the earliest version of android this application can
+# run on. Targeting Android 16 (Jellybean) this that support 99% of current android devices
+PROP_MIN_SDK_VERSION=16
+
+# List of CPU Archtexture to build that application with
+# Available architextures (armeabi | armeabi-v7a | arm64-v8a | x86)
+# To build for multiple architexture, use the `:` between them
+# Example - PROP_APP_ABI=armeabi-v7a:arm64-v8a:x86
 PROP_APP_ABI=armeabi
```

Comment:
Thank you for great update!!!!
Could you also add parameter for generating apk-splits? (seperate apk for each CPU architecture instead of one)
https://developer.android.com/studio/build/configure-apk-splits.html

Reply:
Yea we can, if the cocos team wants it. I currently do it in my own personal project so depending on how they way to do it we can.

It would look something like this:

```
import com.android.build.OutputFile

        versionCode 10
        versionName "0.5.0"
...

       splits {
            abi {
                enable true
                reset()
                include 'armeabi-v7a', 'arm64-v8a', 'x86' /// Would use `PROP_APP_ABI` instead
                universalApk true
            }
        }

        android.applicationVariants.all { variant ->
            // assign different version code for each output
            variant.outputs.each { output ->
                int abiVersionCode = project.ext.abiCodes.get(output.getFilter(OutputFile.ABI)) ?: 0
                output.versionCodeOverride = (abiVersionCode * 100) + android.defaultConfig.versionCode
            }
        }
```

So gradle would produce something like:
 * game-armeabi-v7a | Version 100
 * game-arm64-v8a   | Version 200
 * x86                          | Version 300

Reply:
You are the best! 
Could you create separate pull request for that?
Than we will have smaller code review and changes will be integrated faster into master.
One comment could you use bigger const? maybe 10000 instead of 100?

Reply:
Yea. That is just a sample. First we should ask @minggo and @dumganhar what they think.

Also, 10000? How many updates to your game are you going to publish?! 😂 . Maybe 1000 is reasonable?

Reply:
I think that pull request with passing continuous-integration is the best point for asking questions :)

I already have games with more than 100 updates :)

Reply:
Yea true. This wont pass until the update travis-ci to support ndk16 and AndroidBuildTools 27.0.1

Reply:
Sorry, i don't quite understand what's the issue? Did you mean build for all CPU architecture? If so, then i think it is better not. Because it takes a long time to compile, and will exceed the travis time limitation too.

Reply:
There will be separate pull reqest for that. 
It will work as before.
 But when someone would like to support more than one CPU architecture than there will be created separate apks for each + one apk with all archtectures.

Reply:
K, got it. It is good to have this feature, but should not build all CPU architectures by default.

Reply:
I will add this feature when when we commit this one\